<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>748</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>381</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="layoutPlotIn"/>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>0</y>
      <width>381</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="layoutPlotFFT"/>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>330</y>
      <width>381</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="layoutPlotFFT_phase"/>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>391</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>интенсивность</string>
    </property>
    <widget class="QCheckBox" name="checkBox_CKO">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>30</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>ошибка СКО
или корень</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="spinBox_radius_in">
     <property name="geometry">
      <rect>
       <x>156</x>
       <y>60</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
     <property name="maximum">
      <double>999999.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="spinBox_radius_our">
     <property name="geometry">
      <rect>
       <x>156</x>
       <y>30</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
     <property name="maximum">
      <double>999999.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="spinBox_center_x">
     <property name="geometry">
      <rect>
       <x>91</x>
       <y>90</y>
       <width>66</width>
       <height>24</height>
      </rect>
     </property>
     <property name="maximum">
      <double>999999.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.500000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>64</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Внутренний радиус:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonIntegrate">
     <property name="geometry">
      <rect>
       <x>289</x>
       <y>90</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Интегралить</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="spinBox_center_y">
     <property name="geometry">
      <rect>
       <x>171</x>
       <y>90</y>
       <width>66</width>
       <height>24</height>
      </rect>
     </property>
     <property name="maximum">
      <double>999999.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.500000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>95</y>
       <width>59</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Центр:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>34</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Внешний радиус:</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>&amp;Файл</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_Open"/>
    <addaction name="action_openFFT"/>
    <addaction name="actionOpenImage"/>
    <addaction name="actionOpenImageFFT"/>
    <addaction name="separator"/>
    <addaction name="action_Close"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Open"/>
   <addaction name="actionOpenImage"/>
   <addaction name="action_openFFT"/>
   <addaction name="actionOpenImageFFT"/>
   <addaction name="actionScale"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Открыть</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="text">
    <string>Вы&amp;ход</string>
   </property>
  </action>
  <action name="action_openFFT">
   <property name="text">
    <string>Открыть Фурье</string>
   </property>
  </action>
  <action name="actionScale">
   <property name="text">
    <string>авт. масштаб</string>
   </property>
  </action>
  <action name="actionOpenImage">
   <property name="text">
    <string>Открыть фотку</string>
   </property>
  </action>
  <action name="actionOpenImageFFT">
   <property name="text">
    <string>Открыть фотку Фурье</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
